app/assets/src/components/common/MetadataManualInput.jsx

```diff
@@ -25,12 +25,53 @@ import PropTypes from "~/components/utils/propTypes";
 import PlusIcon from "~ui/icons/PlusIcon";
 import { UserContext } from "~/components/common/UserContext";
 import HostOrganismSearchBox from "~/components/common/HostOrganismSearchBox";
+import ColumnHeaderTooltip from "~/components/ui/containers/ColumnHeaderTooltip";
 
 import cs from "./metadata_manual_input.scss";
 import MetadataInput from "./MetadataInput";
 
 const map = _fp.map.convert({ cap: false });
 
+// From https://czi.quip.com/FPnbATvWSIIL/Metadata-Tooltips#AQKACA1SEBr on 2020-02-18.
+// See also descriptions stored in database by MetadataField.
+// NOTE: for good layout, text should be no longer than 110 chars.
+const COLUMN_HEADER_TOOLTIPS = {
+  "Host Organism": "Host from which the sample was originally collected.",
+  collection_date:
+    "Date on which sample was originally collected. For privacy reasons, only use month and/or year for human data.",
+  collection_location_v2:
+    "Location from which sample was originally collected. For privacy, we do not allow city-level data for human hosts.",
+  nucleotide_type: "Nucleotide type of sample.",
+  sample_type:
+    "Tissue or site from which the sample was originally collected. Suggested list is dependent on host selection.",
+  water_control: "Whether or not sample is a water control.",
+  collected_by: "Institution/agency that collected sample.",
+  isolate: "Whether or not sample is an isolate.",
+  antibiotic_administered: "Antibiotics administered to host.",
+  comorbidity: "Other chronic diseases present.",
+  host_age: "Age of host (in years).",
+  host_genus_species: "Genus or species of host.",
+  host_id: "Unique identifier for host.",
+  host_race_ethnicity: "Race and-or ethnicity of host.",
+  host_sex: "Sex of host.",
+  immunocomp: "Immunocompromised Information on if host was immunocompromised.",
```

Comment:
I'm guessing L57 was just meant to start at "Information" ?

Reply:
Good catch! 😄